views/question.ejs

```diff
@@ -70,8 +70,18 @@
                     </div>
 
                     <!-- Submit Button -->
-                    <button type="submit" class="btn btn-primary col-md-3 mx-auto">Volgende vraag <i
-                            class="bi bi-chevron-right"></i></button>
+                    <% if (typeOfQuiz === "tenrounds") { %>
+                        <% if (questionId === 9) { %>
+                            <button type="submit" class="btn btn-primary col-md-3 mx-auto">Beëindig quiz <i
+                                class="bi bi-chevron-right"></i></button>
+                        <% } %>
+                        <% else { %>
+                            <button type="submit" class="btn btn-primary col-md-3 mx-auto">Volgende vraag <i
+                                class="bi bi-chevron-right"></i></button>
+                        <% } %>
+                    <% } %>
+                    <button type="submit" class="btn btn-primary col-md-3 mx-auto">Beëindig quiz <i
+                        class="bi bi-chevron-right"></i></button>
```

Comment:
Doesn't work indeed. It's much easier to do this with a ternary operator as well. I propose: 

```suggestion
                                        <!-- Submit Button -->
                    <button type="submit" class="btn btn-primary col-md-3 mx-auto">
                        <%= (typeOfQuiz === "tenrounds" && questionId+1 === 10) ? "Beëindig quiz" : "Volgende vraag" %>
                        <i class="bi bi-chevron-right"></i>
                    </button>
```

So basically you keep one singular button, like it was before. And then in the text of the button, you write this little piece of JS:

`<%= (typeOfQuiz === "tenrounds" && questionId+1 === 10) ? "Beëindig quiz" : "Volgende vraag" %>`

This says: if the typeOfQuiz is tenrounds, and the questionId+1 is 10, then the text will say "Beëindig quiz". If not, then it will say "Volgende vraag".

This way it does work.